script/setup/install-dev-tools

```diff
@@ -21,7 +21,7 @@
 set -eu -o pipefail
 
 # install `protobuild` and other commands
-go install github.com/stevvooe/protobuild@v0.1.0
+go install github.com/containerd/protobuild@7e5ee24bc1f70e9e289fef15e2631eb3491320bf
```

Comment:
While protobuild wouldn't need any more changes for this migration, personally I'd like to cut a version once we finish #6564.